### PR TITLE
Updates for 7.1.2 preview release

### PR DESCRIPTION
**Pull request by lilgreenbird**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.1.2] Preview Release
+### Added
+- Added support for JDK 11
+- Added support for Jacoco in VSTS
```
Comment (rene-ye) [inline CHANGELOG.md]: Maybe change to "Updated Maven dependencies, library versions, and Jacoco for Java 11"? That would also move it to the "Changed" section.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
...
+- Updated SQL keywrods in DatabaseMetaData
+- Enabled non-running junit tests
```
Comment (ulvii) [inline CHANGELOG.md]: JUnit

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
...
+- Improvements in Database metadata to prevent Statement leaks and enhance Statement caching
+
+### Fixed Issues
+- Fixed slf4j warning message in tests
+- Fixed NullPointerException in logException
+- Fixed intermittent junit lobTest failures
+- Fixed batch insert in Bulk Copy API to respect timeout value
+
+### Changed
+- Updated junit tests to remove hardcoded names
```
Comment (ulvii) [inline CHANGELOG.md]: JUnit

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
...
+- Updated JDK for Travis CI
```
Comment (ulvii) [inline CHANGELOG.md]: This is already mentioned as `Added support for JDK 11`

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.1.2] Preview Release
+### Added
+- Added support for JDK 11
```
Comment (peterbae) [inline CHANGELOG.md]: For all of these entries, you need to link the PR that we merged. Refer to what we did for 7.1.1 release.

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
...
+- Added support for Jacoco in VSTS
+- Updated SQL keywrods in DatabaseMetaData
+- Enabled non-running junit tests
+- Improvements in Database metadata to prevent Statement leaks and enhance Statement caching
+
+### Fixed Issues
+- Fixed slf4j warning message in tests
+- Fixed NullPointerException in logException
+- Fixed intermittent junit lobTest failures
```
Comment (rene-ye) [inline CHANGELOG.md]: JUnit

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
...
+### Added
+- Added support for JDK 11
+- Added support for Jacoco in VSTS
+- Updated SQL keywrods in DatabaseMetaData
+- Enabled non-running junit tests
+- Improvements in Database metadata to prevent Statement leaks and enhance Statement caching
+
+### Fixed Issues
+- Fixed slf4j warning message in tests
+- Fixed NullPointerException in logException
```
Comment (ulvii) [inline CHANGELOG.md]: Fixed potential NullPointerException in logException()

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
...
+- Fixed intermittent junit lobTest failures
+- Fixed batch insert in Bulk Copy API to respect timeout value
+
+### Changed
+- Updated junit tests to remove hardcoded names
+- Updated JDK for Travis CI
+- Removed illegal reflection access in Kerberos Authentication
+- Updated to use StandardCharsets.US_ASCII instead of hardcoded string
```
Comment (rene-ye) [inline CHANGELOG.md]: Updated Clobs to use StandardCharsets.US_ASCII instead of hard-coded string

File: CHANGELOG.md [hunk 9]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.1.2] Preview Release
+### Added
+- Added support for JDK 11
+- Added support for Jacoco in VSTS
+- Updated SQL keywrods in DatabaseMetaData
+- Enabled non-running junit tests
+- Improvements in Database metadata to prevent Statement leaks and enhance Statement caching
```
Comment (ulvii) [inline CHANGELOG.md]: Please either use `Database Metadata` or `database metadata`

File: CHANGELOG.md [hunk 10]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
...
+
+### Fixed Issues
+- Fixed slf4j warning message in tests
+- Fixed NullPointerException in logException
+- Fixed intermittent junit lobTest failures
+- Fixed batch insert in Bulk Copy API to respect timeout value
```
Comment (ulvii) [inline CHANGELOG.md]: @peterbae , is this accurate?

Comment (peterbae) [reply CHANGELOG.md]: Sounds a bit wrong. I always make my PR title representative of the change (you can mostly copy-paste the PR title to the changelog, and it'll make sense: https://github.com/Microsoft/mssql-jdbc/pull/817)

I would change this to "Fixed Bulk Copy API for batch insert to respect the timeout value".

**Pull request merged**